src/navbar/menu.styles.tsx

```diff
@@ -0,0 +1,207 @@
+import styled from "styled-components";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import { Text, TextStyleHelper } from "../text";
+
+// =============================================================================
+// STYLE INTERFACE, transient props are denoted with $
+// See more https://styled-components.com/docs/api#transient-props
+// =============================================================================
+interface StyleProps {
+    $selected: boolean;
+}
+
+// =============================================================================
+// WRAPPER
+// =============================================================================
+
+export const Wrapper = styled.ul`
+    display: flex;
+    list-style: none;
+    position: relative;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        display: none;
+    }
+`;
+
+export const MobileWrapper = styled.ul`
+    display: none;
+    list-style: none;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        border-left: 4px solid ${Color.Primary};
+        display: flex;
+        flex-direction: column;
+        padding-bottom: 1rem;
+    }
+`;
+
+// =============================================================================
+// LINK ITEMS
+// =============================================================================
+export const LinkItem = styled.li`
+    position: relative;
+    display: flex;
+    align-items: center;
+    :not(:last-of-type) {
+        margin-right: 1rem;
+    }
+
+    ${MediaQuery.MaxWidth.tablet} {
+        width: 100%;
+
+        :not(:last-of-type) {
+            margin-right: 0;
+        }
+    }
+`;
+
+export const Link = styled(Text.Hyperlink.Small)<StyleProps>`
```

Comment:
there are a lot of redundant css styles here. for example line-clamps are the same on desktop and tablet, so there's no need to repeat them again

similarly, `TextStyleHelper` already covers font-size, font-weight, line-height and letter-spacing properties, so there's no need to specify them again